examples/docker/modin-ray/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
is it possible to inherit one dockerfile from another to reduce the copy-pastiness?..

Reply:
This legal notice is mandatory in all files anyway so it has to be copied.

Reply:
It's less than a third, and almost everything else is copied....